R/type-data-frame.R

```diff
@@ -60,7 +60,12 @@ vec_ptype_full.data.frame <- function(x, ...) {
 
 #' @export
 vec_ptype_abbr.data.frame <- function(x, ...) {
-  paste0("df", vec_ptype_shape(x))
+  if (inherits_only(x, "data.frame")) {
+    abbr <- "df"
```

Comment:
Maybe this should be `data.frame`?

Reply:
Is this a bit too long for an abbreviation? I think the idea for this generic was to be the vctrs version of pillar's `type_sum()`.

Reply:
Oh yeah, leave as is and we'll reconsider when we think about what these functions really should be called.